enterprise/app/models/captain/assistant.rb

```diff
@@ -18,6 +18,7 @@
 #
 class Captain::Assistant < ApplicationRecord
   include Avatarable
+  include CaptainToolsHelpers
```

Comment:
lets include with the namespace .. would make it easier to identify where the file is located